livekit-rtc/livekit/rtc/participant.py

```diff
@@ -117,6 +131,42 @@ async def publish_data(
         if cb.publish_data.error:
             raise PublishDataError(cb.publish_data.error)
 
+    async def publish_transcription(
+        self,
+        participant_identity: str,
```

Comment:
I would be more comfortable creating a Transcription structure here.
On the agents API, we will need to return a Transcription, and I don't think this should be defined inside agents.

publish_transcription accepting a "Transcription" seems OK to me